lib/practice_exercises.rb

```diff
@@ -1,13 +1,44 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  item_list = []
+  list.each_with_index do |item|
+    if item_list.include?(item) == false
```

Comment:
Since `.include?` has to traverse the entire array it should take O(n) time, and since it's nested into a loop the method is an O(n<sup>2</sup>) algorithm.